static/src/javascripts/projects/common/modules/commercial/adfree/update-user-features.js

```diff
@@ -0,0 +1,33 @@
+define([
```

Comment:
I tend to agree; I'd only really split it for convenience in testing. For the moment though, I'll bundle it together until a better structure becomes more obvious.